src/biogeochem/CNFireLi2021Mod.F90

```diff
@@ -522,21 +525,18 @@ subroutine CNFireArea (this, bounds, num_soilc, filter_soilc, num_soilp, filter_
            hdmlf = this%forc_hdm(g)
 
            ! calculate human density impact on ag. fire
-           fhd = 0.04_r8+0.96_r8*exp(-1._r8*SHR_CONST_PI*(hdmlf/350._r8)**0.5_r8)
+           fhd = 0.2_r8+0.8_r8*exp(-1._r8*SHR_CONST_PI*(hdmlf/400._r8)) 
 
            ! calculate impact of GDP on ag. fire
-           fgdp = 0.01_r8+0.99_r8*exp(-1._r8*SHR_CONST_PI*(gdp_lf(c)/10._r8))
+           fgdp = 0.05_r8+0.95_r8*exp(-1._r8*SHR_CONST_PI*(gdp_lf(c)/20._r8))
 
            ! calculate burned area
-           fb   = max(0.0_r8,min(1.0_r8,(fuelc_crop(c)-lfuel)/(ufuel-lfuel)))
-
-           ! crop fire only for generic crop types at this time
-           ! managed crops are treated as grasses if crop model is turned on
-           baf_crop(c) = baf_crop(c) + cropfire_a1/secsphr*fhd*fgdp*patch%wtcol(p)
-           if( fb*fhd*fgdp*patch%wtcol(p)  >  0._r8)then
-              burndate(p)=kda
-           end if
-        end if
+            if((use_crop .and. (.not. croplive(p))) &
+                   .or. (.not. use_crop)) then
+                burndate(p) = kda
+                baf_crop(c) = baf_crop(c)+0.21_r8 / secsphr * fhd * fgdp * patch%wtcol(p)
```

Comment:
- [ ] `cropfire_a1` should be set to 0.21 by default and used here.

Reply:
- [ ] Add a `li2024gswpfrc` `fire_method` with this (and other changes that will come in other PRs).